backends/core/workers.py

```diff
@@ -758,3 +761,76 @@ def _execute(self):
     self.retry(request.execute)()
     job_name = '%s/jobs/%s' % (project_id, self._ml_job_id)
     self._enqueue('MLWaiter', {'job_name': job_name}, 60)
+
+
+class MeasurementProtocolWorker(Worker):
+  """Abstract Measurement Protocol worker."""
+
+  def _send_event_hit(self, user_agent='CRMint / 0.1', **kwargs):
+    """Send a measurement protocol hit.
+
+    Arguments:
+      user_agent string representing the client User Agent.
+      kwargs dictionary of key/values to pass to the Measurement
+          Protocol endpoint.
+
+    Raises: WorkerException if the HTTP request fails.
+    """
+    headers = {'user-agent': user_agent}
+    payload = {'v': 1}
+    payload.update(kwargs)
+
+    req = requests.post('https://www.google-analytics.com/collect',
+                        headers=headers,
+                        data=payload)
+
+    if req.status_code != requests.codes.ok:
+      raise WorkerException('Failed to send event hit with status code (%s) '
+                            'and parameters: %s' % (req.status_code, kwargs))
+
+
+class BQToMeasurementProtocol(MeasurementProtocolWorker):
+  """Worker to import data through Measurement Protocol"""
+
+  PARAMS = [
+      ('bq_project_id', 'string', False, '', 'BQ Project ID'),
+      ('bq_dataset_id', 'string', True, '', 'BQ Dataset ID'),
+      ('bq_table_id', 'string', True, '', 'BQ Table ID'),
+  ]
+
+  def _get_client(self):
```

Comment:
I didn't inherit from `BQWorker` class, because I don't want to use `_bq_setup()` the [way it is](https://github.com/google/crmint/blob/dev/backends/core/workers.py#L166-L171).

I can maybe suggest a refactoring, because the `_bq_setup()` should actually belong to the `BQQueryLauncher` class.